charts/flyte-agent/values.yaml

```diff
@@ -0,0 +1,77 @@
+# ---------------------------------------------------------------------
```

Comment:
Will there be value in consolidating the format of our `values.yaml` files so that users can start getting familiar. Right now, all charts are very different from each other.